conf/application.conf

```diff
@@ -103,7 +103,7 @@ metrics {
   durationUnit = SECONDS
   showSamples = true
   jvm = true
-  enabled = false
```

Comment:
did you mean to turn metrics on? If so, fine just wanted to check

Reply:
yes, this means we will collect metrics (vs graphite which is the means of transferring the data). Graphite is already covered in higher environments but metrics is not.